channeldb/channel.go

```diff
@@ -518,6 +522,36 @@ func (c *OpenChannel) MarkAsOpen(openLoc lnwire.ShortChannelID) error {
 	})
 }
 
+// MarkBorked marks the event when the remote party's state has reached an
+// irreconcilable state.
```

Comment:
It is not really clear to me when a state is "irreconcilable" or "desynchronized". Could you add a comment explaining in which cases this will be marked "borked", and what will happen to a borked channel?